src/archetypes/referencebrowserwidget/browser/view.py

```diff
@@ -282,18 +282,38 @@ def getResult(self):
             self.request.form['path'] = {
                 'query': '/'.join(folder.getPhysicalPath()),
                 'depth': 1}
+            self.request.form['portal_type'] = []
             if 'sort_on' in self.widget.base_query:
                 self.request.form['sort_on'] = self.widget.base_query['sort_on']
             else:
                 self.request.form['sort_on'] = 'getObjPositionInParent'
 
             result = qc(search_catalog=self.widget.search_catalog)
+
         else:
             result = []
+
         b_size = int(self.request.get('b_size', 20))
         b_start = int(self.request.get('b_start', 0))
-
-        return Batch(result, b_size, b_start, orphan=1)
+        return Batch(self._prepareResults(result), b_size, b_start, orphan=1)
```

Comment:
@esteele do you think this change requires a 2.6 release rather than 2.5.3 or is the information in the changelog enough?
(sorry for asking again, but i f*** up my previous PR when trying `git rebase` to make it mergeable again)